src/Binah/JSON.hs

```diff
@@ -64,7 +81,14 @@ decodeBody = do
     Left  s -> respondError status400 (Just s)
     Right a -> return a
 
-
+{-@ decodeFiles :: TaggedT<{\_ -> True}, {\v -> v == currentUser 0}> _ _ _ @-}
+decodeFiles :: (MonadTIO m, MonadController TIO m) => TaggedT user m ([Param], [File LBS.ByteString]) 
+decodeFiles = do
+  req <- requestT
+  liftTIO   $ TIO $ parseRequestBodyEx opts lbsBackEnd (waiRequest req)
```

Comment:
A note on soundness. This may look like is verified, but `TIO` is an unverified operation and it shouldn't be allowed in client code. So, this function is implicitly marking `parseRequestBodyEx` as a safe function. Maybe a better way to do this is to expose 
```haskell
parseRequestBodyExT opts backend r = TIO $ parseRequestBodyEx opts (unRequestTIO r)
```
from `Binah.Frankie` instead of exposing `unRequestTIO` directly